components/Search/Search.tsx

```diff
@@ -1,109 +1,148 @@
-import { useEffect, ComponentProps } from "react";
+import React, { useEffect, useRef, createElement, Fragment } from "react";
+import { autocomplete } from "@algolia/autocomplete-js";
+import "@algolia/autocomplete-theme-classic";
+import { render } from "react-dom";
 import styled from "styled-components";
-import { css, wrapper, StyledSystemWrapperProps } from "components/system";
-import Icon from "components/Icon";
+import css from "@styled-system/css";
+import Box from "components/Box";
+import { debounced } from "server/search-api-helpers";
 import { SearchStyles } from "./SearchStyles";
-import docVersions from "../../config.json";
 
-export interface SearchProps {
-  id?: string;
-  version?: string;
-}
+export const getSearchResults = async (query: string) => {
+  let href = "";
+  if (typeof window !== "undefined") {
+    href = window.location.href;
+  }
+
+  try {
+    const rawResponse = await fetch(
+      `/docs/api/search/?query=${query}&url=${encodeURIComponent(href)}`,
```

Comment:
I think we don't use `url` anywhere on the server, we only use `query`. Instead of `url` we use referrer there.

Also we an get current version from the scope here and send it directly, instead of parsing urls to simplify things.